test/docfx.Test/DocfxTest.cs

```diff
@@ -247,6 +275,21 @@ private static JsonDiff CreateJsonDiff()
                 .Build();
         }
 
+        private static JsonDiff CreateLanguageServerNotificationJsonDiff()
+        {
+            var fileJsonDiff = new JsonDiffBuilder()
```

Comment:
This is currently enough to verify response payload json

```suggestion
           return new JsonDiffBuilder()
```